package.json

```diff
@@ -98,10 +98,12 @@
     "karma-browserstack-launcher": "^1.6.0",
     "karma-chrome-launcher": "^3.1.0",
     "karma-coverage": "^2.0.1",
+    "karma-firefox-launcher": "^2.1.0",
     "karma-jasmine": "^2.0.1",
     "karma-jasmine-html-reporter": "^1.4.2",
     "karma-jasmine-matchers": "^4.0.1",
     "karma-rollup-preprocessor": "^7.0.5",
+    "karma-safari-launcher": "^1.0.0",
```

Comment:
I added these launchers to make it easier for us to test on Firefox and Safari locally.